src/main/java/com/danielagapov/spawn/DTOs/FriendRequestDTO.java

```diff
@@ -0,0 +1,9 @@
+package com.danielagapov.spawn.DTOs;
+
+import com.danielagapov.spawn.Models.User;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+public record FriendRequestDTO(UUID id, User sender, User receiver) implements Serializable {
```

Comment:
Hey, looks great! Just one thing, could you switch these `User` objects to `UserDTO`? Apart from that, everything else looks completely solid, so I'll approve right afterwards.